integration/src/main/java/scripts/testscripts/SnapshotDatasetOperations.java

```diff
@@ -53,7 +54,13 @@ public class SnapshotDatasetOperations extends TestScript {
 
   @Override
   public void setup(List<TestUserSpecification> testUsers) throws Exception {
-    DatarepoClient datarepoClient = new DatarepoClient(server, testUsers.get(0));
+    DatarepoClient datarepoClient = null;
+    for (TestUserSpecification testUser : testUsers) {
+      if (testUser.userEmail.equals(ADMIN_EMAIL)) {
+        datarepoClient = new DatarepoClient(server, testUser);
+      }
+    }
+    assertNotNull(datarepoClient);
```

Comment:
This change is unnecessary as this test only has a single element in `testUsers` and it's always the admin user. If we need to add more users in the future, we can update the code then to handle this case.
```suggestion
    DatarepoClient datarepoClient = new DatarepoClient(server, testUsers.get(0));
 ```